tests/test_flow.py

```diff
@@ -225,9 +224,8 @@ async def async_offset_commit():
     try:
         assert offsets == {("/", i): num_records_per_shard for i in range(num_shards)}
     finally:
-        await controller.terminate()
+        termination_result = await controller.terminate(wait=True)
 
-    termination_result = await controller.await_termination()
```

Comment:
why this line is removed?

Reply:
Replaced by `termination_result = await controller.terminate(wait=True)` which terminates and awaits termination in one.